4week/src/components/Sidebar.tsx

```
@@ -1,88 +1,148 @@
+                  className="hover:text-pink-400 flex items-center gap-2"
+                >
+                  찾기
+                </Link>
+                <Link
+                  to="/my"
+                  className="hover:text-pink-400 flex items-center gap-2"
+                >
+                  마이페이지
+                </Link>
+
+                <button
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)